threejs/fxhash_Rooms/src/index.js

```diff
@@ -141,26 +129,10 @@ for (let i = 0; i < figures.length; i++) {
       figures[i].scale.z
     );
   }
-  /*
-  groundPlaneMesh = new THREE.Mesh(groundPlaneGeometry, material);
-  groundPlaneMesh.castShadow = true;
-  groundPlaneMesh.receiveShadow = true;
-  groundPlaneMesh.position.set(
-    figures[i].pos.x,
-    figures[i].pos.y,
-    figures[i].pos.z
-  );
-  groundPlaneMesh.rotation.set(
-    figures[i].rot.x,
-    figures[i].rot.y,
-    figures[i].rot.z
-  );
-    groundPlaneMesh.scale.set(
-    figures[i].scale.x,
-    figures[i].scale.y,
-    figures[i].scale.z
-  );
-*/
+
+  roomMeshOutlines = new THREE.LineSegments(new THREE.EdgesGeometry(roomMesh.geometry), new THREE.LineBasicMaterial({color:"black"}));
```

Comment:
Hopefully a super quick one for you @Anaglyph-ic 

Any recommened approach to adding outlines in what you've previously done? This method produces the outlines I'm looking for but still operates on the pre-traslated geometry. 

![image](https://user-images.githubusercontent.com/54593392/185757599-c8d15400-2a12-485e-bffb-faf39c3ce391.png)

Reply:
I forgot I had to apply the same translation to outline geometry/material as well.
```  roomMeshOutlines = new THREE.LineSegments(new THREE.EdgesGeometry(roomMesh.geometry), new THREE.LineBasicMaterial({color:"black"}));
  roomMeshOutlines.position.set(figures[i].pos.x, figures[i].pos.y, figures[i].pos.z);
  roomMeshOutlines.rotation.set(figures[i].rot.x, figures[i].rot.y, figures[i].rot.z);
  if (figures[i].hasOwnProperty("scale")) {
    roomMeshOutlines.scale.set(
      figures[i].scale.x,
      figures[i].scale.y,
      figures[i].scale.z
    );
  }